tests/unit_tests/test_wave_03.py

```diff
@@ -38,9 +38,9 @@ def test_swap_items_returns_true():
     assert item_d not in jolie.inventory
     assert item_e in jolie.inventory
     assert item_b in jolie.inventory
-    assert result
+    assert result == True
```

Comment:
Remember that `assert result` and `assert result == True` do exhibit the same behavior!